### DISCUSSION TITLE
Can't map to Backspace

### DISCUSSION THREAD

**Discussion by omerchn** [evan-liu/karabiner.ts#85] [Q&A] [answered]
Thanks in advanced for the help!
So basically, I want to map fn+backspace to option+backspace.
Dug around in the types and found that the symbol ⌫ represents backspace.
But when I try to use it, it doesn't recognize and i get a type error, and also when I build, it doesn't seem to work.
That is my code currently:
```ts
  rule('fn delete word').manipulators([
    // config key mappings
    map('fn', '⌫').to('right_option', '⌫'),
  ]),
```
How would I go about doing this?

**Comment by evan-liu** [comment, marked as answer]
Hi @omercohen990 , thanks for trying out the tool. Sorry for the late reply as I somehow didn't get emailed by GitHub.

It can be one of these: 

```typescript
map('⌫', 'fn').to('⌫', 'right_option')
map('⌫', 'fn').to('⌫', '›⌃')
map('⌫', 'fn').to('⌫', '⌃') // side normally doesn't matter as modifiers 
map('delete_or_backspace', 'fn').to('delete_or_backspace', 'right_option')
```

See more here: https://evan-liu.github.io/karabiner.ts/manipulators/from#map-key

**Reply by evan-liu** [reply]
[🛝 Try it here](https://stackblitz.com/edit/karabiner-ts-omercohen990?file=rules.js)

**Reply by omerchn** [reply]
It works, tyvm!!
